index.html

```diff
@@ -240,5 +240,34 @@ <h3>Skills</h3>
         </li>
       </ul>
     </section>
+
+    <section class="form-container">
+         <div class="form-text">
+           <h1>Contact me</h1>
+         </div>  
+      <div class="paragragh">
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      </div>
+      
+      <div class="contact-form">
+        <form action="https://formspree.io/f/xzboload" method="post" name="form-collection" class="my-form-collection">
+          <ul>
+            <li>
+                <input type="text" placeholder="benjamin" maxlength="30" required>
+            </li>
+  
+            <li>
+                <input type="text" placeholder="Ben@gmail.com" required>
+            </li>
+  
+            <li>
+                <textarea name="text" cols="30" rows="10" placeholder="write your massage here" maxlength="500"
+                required></textarea>
+            </li>
+          </ul>
+          <button type="submit" class="contact-button">Get in touch</button>
```

Comment:
- [ ] I think here the inputs heigh is not similar to the Figma template kindly Fix them to be the same as the Figma design according to the requirements.

- [ ] As I can see here also the background is not aligned well please add it to be the same

- Your design
![image](https://user-images.githubusercontent.com/58553711/179063825-91fe91d9-7f94-4a8a-99fe-abf0ac9d46c2.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179063976-b75561d9-87d2-482e-9d3d-49bcf0d5eb44.png)